app/v1/connectors/TriggerBsasConnector.scala

```diff
@@ -20,8 +20,8 @@ import javax.inject.{Inject, Singleton}
 import config.AppConfig
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
-import v1.models.request.TriggerBsasRequestData
-import v1.models.response.triggerBsas.TriggerBsasResponse
+import v1.models.request.triggerBsas.TriggerBsasRequestData
+import v1.models.response.TriggerBsasResponse
```

Comment:
better keep a separate package for the response also... if the response has many sub-elements then we can keep them in the same package